tests/wicked/aggregate/ref/t12_teaming_random.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Teaming, Random
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+
+use Mojo::Base 'wickedbase';
```

Comment:
Now we are using `Mojo::Base` for inheritance? I like it :+1: but wont to point @asmorodskyi to it.

Reply:
I wonder if it's really needed in REF tests with empty code :)

Reply:
@cfconrad  I just give up :)
@jlausuch I think we could drop a lot of ref files in a fact . we just need to come up with smarter logic which would call load_test function against same parameterized file

Reply:
I think for the empty code thingy... we could just add a "common_ref and load this one.